README.md

```diff
@@ -16,11 +16,21 @@
 [![Total visitor](https://visitor-count-badge.herokuapp.com/total.svg?repo_id=vald)](https://github.com/vdaas/vald/graphs/traffic)
 [![Visitors in today](https://visitor-count-badge.herokuapp.com/today.svg?repo_id=vald)](https://github.com/vdaas/vald/graphs/traffic)
 
-vald is high scalable distributed high-speed approximate nearest neighbor search engine
+<div align="center">
+Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine.<br>
+Vald is designed and implemented based on the Cloud-Native architecture, <br>
+and enables automatic vector indexing and index backup, and horizontal scaling which made for searching from billions of feature vector data.<br>
```

Comment:
hmm, we can describe these things using your paragraph.
but we also need sentence paragraph like [k8s README.md](https://github.com/kubernetes/kubernetes)

Reply:
I think it is a little bit different... this is the feature list but the k8s one is the background... I think listing the feature instead of writing a paragraph is easier to understand..

Reply:
I think both of them is important we shoud write both,

Reply:
In about section we need to describe background of vald `why vald`